src/components/Header/components/NavButtonsRightCorner/index.jsx

```diff
@@ -80,7 +80,7 @@ const GordonNavButtonsRightCorner = ({ onClose, openDialogBox, open, anchorEl })
     isAuthenticated && isSiteAdmin ? (
       <GordonNavButton
         unavailable={!isOnline ? 'offline' : null}
-        onLinkClick={onClose}
+        zonLinkClick={onClose}
```

Comment:
Is this a mistype? I can't find anything referencing that property :)

Reply:
I just reverted the error code and re-committed it.